### PR TITLE
Add old news posts

### PR DESCRIPTION
**Pull request by zg**
I've gone through and added in old news posts from across the years that were missing from the website.

Some notes worth mentioning:

- Any posts before August 2002 are difficult to decipher due to them being unformatted. I'll try and decipher them and add them in at a later date.
- The general meeting that was [announced](http://www.nblug.org/pipermail/announce/2011q2/000236.html) in June 2011 never happened - Frank said he did the presentation on [July 12, 2011](http://www.nblug.org/pipermail/announce/2011q3/000237.html) instead.
- [February 2012 meeting was cancelled](http://www.nblug.org/pipermail/announce/2012q1/000244.html) due to the meeting day falling on Valentine's Day.

### CODE REVIEW

File: content/news/2002-09-11-the-legalities-of-open-source-and-linux.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+
+Larry Rosen is the founding partner of [ROSENLAW.COM LLP](http://www.rosenlaw.com/), a law firm that specializes in intellectual property matters for large and small companies and individuals. He is also the general counsel and executive director of [Open Source Initiative](http://www.opensource.org/), an organization that reviews and approves licenses for open source software. His articles on legal issues relating to open source appear monthly in [Linux Journal](http://www.linuxjournal.com/search.php?author=LawrenceRosen). Before he became an attorney, Larry taught computer programming and managed several computer departments at [Stanford University](http://www.stanford.edu/). He then worked in private industry, managing computer and applications development. At one large firm he guided the design, implementation, manufacturing and marketing of successful products for data and voice communications.
+
+[http://www.opensource.org/](http://www.opensource.org/)
```
Comment (twm) [inline content/news/2002-09-11-the-legalities-of-open-source-and-linux.md]: Could you update links to opensource.org to be https://?

File: content/news/2002-09-11-the-legalities-of-open-source-and-linux.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+[http://www.rosenlaw.com/html/lawrenceerosen.html](http://www.rosenlaw.com/html/lawrencerosen.html)
+
+[http://www.linuxjournal.com/search.php?author=LawrenceRosen](http://www.linuxjournal.com/search.php?author=LawrenceRosen)
```
Comment (twm) [inline content/news/2002-09-11-the-legalities-of-open-source-and-linux.md]: Update to https://www.linuxjournal.com/user/800878 ?

File: content/news/2002-08-13-database-administration-with-mysql-and-postgresql.md [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+Tags: general meeting
+Event: 2002-08-13 7:30 pm to 9:00 pm
+Speaker: Augie Schwer
+Location: O'Reilly and Associates
+Author: Eric Eisenhart
+
+Augie will be covering both PostgreSQL Database Administration and MySQL Database Administration. So if you've just been dying to set up a database or you have one of these databases and aren't quite sure how to configure it or add users or just don't quite know what to do with it, come to the meeting with your brains ready. We're hoping some of you will be coming with questions ready so that Augie can be properly grilled.
+
+MySQL and PostgreSQL are the two leading open source SQL Relational DataBase Management Systems (RDBMS); both run on Linux as well as a variety of other platforms, mostly *nix. They are used to power many websites, including slashdot.org, freshmeat.net, SourceForge.net, quickfacts.census.gov, finance.yahoo.com, mail-abuse.org, and more projects than you can shake a stick at. MySQL is a lightweight database engine that does most of what you want; PostgreSQL is a bit heftier but also provides all of what's in the ANSI SQL92 standard. Both can be used from C, C++, Perl, PHP, Java, ODBC and Python.
```
Comment (twm) [inline content/news/2002-08-13-database-administration-with-mysql-and-postgresql.md]: Could you backslash-escape the `*` in `*nix`?

File: content/news/2003-01-18-installfest.md [hunk 4]
```diff
@@ -0,0 +1,17 @@
+Title: InstallFest
+Tags: installfest
+Event: 2003-01-18 10:30 am to 4:00 pm
+Speaker: None
+Location: Sonoma State Campus Schulz Info Center, Room 1121 (see below for specifics)
+Author: Eric Eisenhart
+
+WHAT: NBLUG InstallFest
```
Comment (twm) [inline content/news/2003-01-18-installfest.md]: I think that we need `<br>` tags or this will run into one paragraph.

Comment (zg) [inline content/news/2003-01-18-installfest.md]: I didn't know `<br>` tags were legal in Markdown - I'll make the appropriate changes right away.

File: content/news/2002-09-11-the-legalities-of-open-source-and-linux.md (hunk 2 above)
Comment (zg) [inline content/news/2002-09-11-the-legalities-of-open-source-and-linux.md]: Sure thing. I simply copied the links, I didn't know if we should update them or not.

File: content/news/2003-09-27-installfest.md [hunk 5]
```diff
@@ -0,0 +1,63 @@
...
+Location: SSU Library, Room 1121
+Author: Dustin Mollo
+
+Details:
+
+Please note that this InstallFest is NOT at O'Reilly and Associates. It will instead be at Sonoma State University in Rohnert Park. Please see below for information on the exact location and directions.
+
+What is an InstallFest and who can come?
+
+Instead of putting all of the usual "what is an InstallFest" info in this and future announcements, we have a link up on our website as to what a NBLUG InstallFest entails and how to participate. If you point your browser of choice to http://www.nblug.org/installfest/, you'll be presented with all the info you ever wanted to know about what goes on at an InstallFest, and how you can get the most out of attending one.
```
Comment (twm) [inline content/news/2003-09-27-installfest.md]: We link to this a lot, but we don't have a page here...

Comment (zg) [inline content/news/2003-09-27-installfest.md]: *shrugs* I copied things over from the Internet Archive. I think we should preserve the URLs for historical purposes, but just not make them clickable. What do you think?

File: README.md [hunk 6]
```diff
@@ -74,7 +74,7 @@ The script ``./add-talk.py`` automatically generates a general meeting news item
 
 When migrating pages from the Drupal site, an additional metadata field can be used to make sure the page shows up with the same URL:
 
-    Druapl_Node: <number>
+    Drupal_Node: <number>
```
Comment (twm) [inline README.md]: :-) Thank you.

File: content/news/2003-09-27-installfest.md (hunk 5 above)
Comment (twm) [inline content/news/2003-09-27-installfest.md]: They automatically become clickable because Markdown, but yeah, *shrug*.  Don't worry about it.

File: content/news/2011-08-09-bitcoin-and-namecoin.md [hunk 7]
```diff
@@ -1,10 +1,9 @@
 Title: Bitcoin and Namecoin
 Tags: general meeting
-Start: 2011-08-09 7:30 pm
-End: 2011-08-09 9:00 pm
+Event: 2011-08-09 7:30 pm to 9:00 pm
 Speaker: Allan Cecil
 Location: O'Reilly Media's Tarsier conference room
-Author: Glenn Kerbein
+Author: Allan Cecil
```
Comment (twm) [inline content/news/2011-08-09-bitcoin-and-namecoin.md]: Don't change this.  The author is the person who posted to the site, which was Glen, not Allan.

File: content/news/2013-06-03-lightning-talks.md [hunk 8]
```diff
@@ -4,6 +4,7 @@ Event: 2013-06-11 7:30 pm to 9:00 pm
 Location: O'Reilly Media
 Date: 2013-06-03 11:07
 Drupal_Node: 192
+Slug: lightning-talk-2013
```
Comment (twm) [inline content/news/2013-06-03-lightning-talks.md]: I think the slug overrides `Drupal_Node`; why did you change this?

**Comment by twm** [comment]
I've scanned through everything.  You can rebase as you wish.  I will merge tomorrow evening..

File: content/news/2013-06-03-lightning-talks.md (hunk 8 above)
Comment (twm) [inline content/news/2013-06-03-lightning-talks.md]: Or maybe I am wrong.  You got warnings from the build about this?  Just make sure that the actual HTML is generated at `/node/192/index.html` and we should be okay.

Comment (zg) [inline content/news/2013-06-03-lightning-talks.md]: `make devserver` was complaining. Let me try and remove the slug and see if it still complains.

**Comment by zg** [comment]
Unless you wanted to look into making sure the InstallFest link isn't clickable, you should be good to go.

**Comment by twm** [comment]
Thanks @zg.  These changes are live on nblug.org now!
